.prettierrc

```diff
@@ -0,0 +1,4 @@
+{
+  "printWidth": 120,
```

Comment:
Prettier team recommends against using print widths > 80 because of how the tool packs characters:

> For readability we recommend against using more than 80 characters:
>
> In code styleguides, maximum line length rules are often set to 100 or 120. However, when humans write code, they don't strive to reach the maximum number of columns on every line. Developers often use whitespace to break up long lines for readability. In practice, the average line length often ends up well below the maximum.
>
> Prettier, on the other hand, strives to fit the most code into every line. With the print width set to 120, prettier may produce overly compact, or otherwise undesirable code.

https://prettier.io/docs/en/options.html

Reply:
thanks for the feedback! i was following the prettier settings that Betterment uses in other projects. i'll rerun with 80 to see what the code looks like.